vulnerabilities/migrations/0005_auto_20220328_1929.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 4.0.2 on 2022-03-28 19:29
+
+import hashlib
+import json
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    def md5hash(apps, schema_editor):
+        Advisory = apps.get_model("vulnerabilities", "Advisory")
+        for advisory in Advisory.objects.all():
+            hash = ""
```

Comment:
Since you can also ``json.dumps()`` a plain string or even a null, what about this simpler loop?
```
checksum = hashlib.md5()
for field in (advisory.summary, advisory.affected_packages, advisory.references):
    value = json.dumps(field, separators=(",", ":")).encode("utf-8")
    checksum.update(value)
advisory.unique_content_id = checksum.hexdigest()
advisory.save()
```

You likely want to use the same below too.